package/network/services/hostapd/files/hostapd.sh

```diff
@@ -1165,7 +1165,7 @@ wpa_supplicant_set_fixed_freq() {
 	append network_data "frequency=$freq" "$N$T"
 	case "$htmode" in
 		NOHT) append network_data "disable_ht=1" "$N$T";;
-		HT20|VHT20) append network_data "disable_ht40=1" "$N$T";;
+		HE20|HT20|VHT20) append network_data "disable_ht40=1" "$N$T";;
 		HT40*|VHT40*|VHT80*|VHT160*) append network_data "ht40=1" "$N$T";;
```

Comment:
Do we have to set `ht40=1` for HE40 HE80 and HE160 too?

Reply:
It seems it is not necessary, the radio stays at the fixed bandwidth indicated in those htmodes.